src/main/java/cachiluis/model/School.java

```diff
@@ -0,0 +1,66 @@
+package cachiluis.model;
+
+import datastructures.arraylist.MyArrayList;
+
+public class School {
+    private String name;
+    private static School school;
```

Comment:
Not against you using a Singleton. Seems appropriate minor comment static are usually put before normal class attributes.